net-misc/curl/curl-7.64.1.recipe

```diff
@@ -7,14 +7,14 @@ based upload, proxies, cookies, user+password authentication (Basic, Digest, \
 NTLM, Negotiate, kerberos...), file transfer resume, proxy tunneling and a \
 busload of other useful tricks."
 HOMEPAGE="https://curl.haxx.se/"
-COPYRIGHT="1996-2018 Daniel Stenberg"
+COPYRIGHT="1996-2019 Daniel Stenberg"
 LICENSE="Curl"
 REVISION="1"
 SOURCE_URI="https://curl.haxx.se/download/curl-$portVersion.tar.bz2"
-CHECKSUM_SHA256="7802c54076500be500b171fde786258579d60547a3a35b8c5a23d8c88e8f9620"
+CHECKSUM_SHA256="4cc7c738b35250d0680f29e93e0820c4cb40035f43514ea3ec8d60322d41a45d"
 ADDITIONAL_FILES="curl.rdef"
 
-ARCHITECTURES="x86_gcc2 x86 x86_64 ?arm ?ppc sparc"
+ARCHITECTURES="x86_gcc2 x86 x86_64 ?arm ?ppc"
```

Comment:
Why?

Reply:
haikuporter said 'not supported'. Maybe works on buildbot(s), but not locally for that arch.
See: m68k,ppc,arm. (Enhancement: Maybe need to patch haikuporter to fix...)

Reply:
Please update your haikuporter, it is out of date.

Reply:
Done.